(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'Rows' object has no attribute 'Title'"
p3
sS'layer'
p4
S'/home/rovyas/Documents/web2py/web2py/applications/pluralsight/controllers/priority.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f8958da04b0, file "/home/rovyas/Documents/web2py/web2py/applications/pluralsight/controllers/priority.py", line 4>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/rovyas/Documents/web2py/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/rovyas/Documents/web2py/web2py/applications/pluralsight/controllers/priority.py", line 232, in <module>\n  File "/home/rovyas/Documents/web2py/web2py/gluon/globals.py", line 414, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/rovyas/Documents/web2py/web2py/applications/pluralsight/controllers/priority.py", line 52, in test\n    print read_db_data.Title\nAttributeError: \'Rows\' object has no attribute \'Title\'\n'
p11
s.